osfoffline/views/rsc/startscreen.py

```diff
@@ -11,43 +11,50 @@
 class Ui_startscreen(object):
     def setupUi(self, startscreen):
         startscreen.setObjectName("startscreen")
-        startscreen.resize(642, 374)
+        startscreen.resize(280, 177)
+        self.gridLayout = QtWidgets.QGridLayout(startscreen)
+        self.gridLayout.setContentsMargins(11, 11, 11, 11)
+        self.gridLayout.setSpacing(6)
+        self.gridLayout.setObjectName("gridLayout")
         self.groupBox_7 = QtWidgets.QGroupBox(startscreen)
-        self.groupBox_7.setGeometry(QtCore.QRect(100, 110, 441, 221))
         self.groupBox_7.setObjectName("groupBox_7")
+        self.gridLayout_2 = QtWidgets.QGridLayout(self.groupBox_7)
+        self.gridLayout_2.setContentsMargins(11, 11, 11, 11)
+        self.gridLayout_2.setSpacing(6)
+        self.gridLayout_2.setObjectName("gridLayout_2")
+        self.passwordEdit = QtWidgets.QLineEdit(self.groupBox_7)
+        self.passwordEdit.setEchoMode(QtWidgets.QLineEdit.Password)
+        self.passwordEdit.setObjectName("passwordEdit")
```

Comment:
It's a small thing, but Al the Octopus thanks you for giving your UI fields human-readable names. :octopus: